### DISCUSSION TITLE
Sample for error handling  .files.add(.. and .files.addChunked(..

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/sp-dev-fx-webparts#1914] [General]
## Sample Description (what should it do)
At run time, my users encounter some errors while uploading files to SharePoint libraries. 
the sample will show how to decide based on the error if it should retry or abort.
example if the error is unauthorize, abort. If the error is network related, retry.

Also, I am looking for advice how to create the errors in the developer tool while developing to be able to react to them in code.

Any advice is welcome and thank you in advance.

**Comment by ghost** [comment]
Thank you for reporting this issue. We will be triaging your incoming issue as soon as possible.

**Comment by hugoabernier** [comment]
This is a great idea for a sample. If anyone from the community wants to help to build this sample, it would be amazing!!!

**Reply by amirmsai** [reply]
Hi @hugoabernier 
if i have the basic code, i can build it.

Amir MSAI
